core/base/src/TObject.cxx

```diff
@@ -158,6 +139,11 @@ TObject::~TObject()
    if (fgObjectStat && gObjectTable) gObjectTable->RemoveQuietly(this);
 }
 
+void TObject::AddToTObjectTable(TObject *op)
```

Comment:
Can do. It is here because I could not use directly TObjectTable::AddObj because this would introduce a cyclic dependency of headers.